parent/pom.xml

```diff
@@ -98,10 +98,6 @@
     </mailingLists>
 
     <properties>
-        <sonatypeOssDistMgmtNexusUrl>https://jakarta.oss.sonatype.org/</sonatypeOssDistMgmtNexusUrl>
```

Comment:
Only now I see that in https://github.com/eclipse-ee4j/ee4j/blob/3f28506c4bfe90db74edfa522606d90e58ed1c1d/parent/pom.xml#L142 `sonatypeOssDistMgmtNexusUrl` is undefined.

Reply:
Indeed. This PR really needs to be reverted ASAP @ivargrimstad 
Thank you